<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Insert</class>
 <widget class="QWidget" name="Insert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layoutGWidget">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageOperation">
      <layout class="QGridLayout" name="layoutGOperation">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="layoutHButton">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <spacer name="opSpacerH">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="opButtonClear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="opButtonClose">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="opButtonUpdate">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="opButtonInsert">
           <property name="text">
            <string>Insert</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="opGroupData">
         <property name="title">
          <string>Data</string>
         </property>
         <layout class="QGridLayout" name="layoutGData">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <property name="spacing">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="opLabelQ">
            <property name="text">
             <string>Question</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="opLabelA">
            <property name="text">
             <string>Answer</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPlainTextEdit" name="opTextA"/>
          </item>
          <item row="1" column="0">
           <widget class="QPlainTextEdit" name="opTextQ"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="opGroupProperty">
         <property name="title">
          <string>Property</string>
         </property>
         <layout class="QGridLayout" name="layoutGProperty">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <property name="spacing">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="opLabelId">
            <property name="text">
             <string>id</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="opLabelType">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="opComboType"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="opLabelLevel">
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="opComboLevel"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="opLabelSubject">
            <property name="text">
             <string>Subject</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="opComboSubject"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="opLabelStatus">
            <property name="text">
             <string>Status</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSpinBox" name="opSpinStatus"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="opLabelTips">
            <property name="text">
             <string>Tips</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="opLabelCata">
            <property name="text">
             <string>Cata</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPlainTextEdit" name="opPlainTips"/>
          </item>
          <item row="4" column="2" colspan="2">
           <widget class="QPlainTextEdit" name="opPlainCata"/>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QLineEdit" name="opLineId"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageDisplay">
      <layout class="QGridLayout" name="layoutGDisplay">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QWebEngineView" name="displayWebview">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="displaySpacerH">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>508</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="displayButtonOk">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
